docs/en/observability/synthetics-run-test.asciidoc

```diff
@@ -0,0 +1,146 @@
+----
+Journey: inline
+   ✓  Step: 'load homepage' succeeded (1831 ms)
+   ✓  Step: 'hover over products menu' succeeded (97 ms)
+
+ 2 passed (2511 ms)
+----
+
+[discrete]
+[[synthetics-inline-journey-send]]
+=== Send to {stack}
+
+After testing locally, you can copy the script into your Elastic Synthetics integration or `heartbeat.yml`
+depending on <<uptime-set-up-choose,the approach you chose>>.
+
+--
+include::tab-widgets/uptime-monitoring/run-test-inline-widget.asciidoc[]
+--
+
+[discrete]
+[[synthetics-test-suite]]
+== Use a test suite
+
+If you have a suite of tests, you can use the `@elastic/synthetics` package as a library.
+
+To test an test suite locally, start by writing your tests:
+
+. Create a new https://docs.npmjs.com/cli/v7/commands/npm-init[NPM/Node.js project].
+. Create a `javascript` or `typescript` file that imports your tests.
+All synthetic test files must use the `.journey.ts` or `.journey.js` file extension.
+. Make sure any dependencies are defined with an `import` outside of the `journey` object.
+. Compile everything together.
+
+To see an example of a test suite, see the https://github.com/elastic/synthetics[elastic/synthetics] repository.
+You can use the `todo` directory if you'd like to test an example locally.
+Clone the repository, change into the directory containing the synthetics test suite,
+and install the necessary dependencies including `@elastic/synthetics`:
+
+[source,sh]
+----
+# Check out the synthetics repo and included examples
+git clone git@github.com:elastic/synthetics.git &&\
+cd synthetics/examples/todos/ &&\
+# Install all required dependencies for the todos example
+npm install
+----
+
+[NOTE]
+==============
+If you want to test your own suite locally, be sure to install the `@elastic/synthetics` package:
+
+[source,sh]
+----
+npm install -g @elastic/synthetics
+----
+==============
+
+From inside the directory containing the synthetics test suite you can run the provided tests.
+By default, the `elastic-synthetics` runner will only run files matching the filename `*.journey.(ts|js)*`.
+
+[source,sh]
+----
+# Run tests on the current directory. The dot `.` indicates
+# that it should run all tests in the current directory.
+npx @elastic/synthetics .
+----
+
+[discrete]
+[[synthetics-test-suite-send]]
+=== Send to {stack}
+
+After testing locally, you can run the test suite using the Elastic Synthetics integration or {heartbeat}
+depending on <<uptime-set-up-choose,the approach you chose>>.
+
+--
+include::tab-widgets/uptime-monitoring/run-test-suite-widget.asciidoc[]
+--
+
+[discrete]
+[[synthetics-ci]]
+== Run on CI
+
+Besides running locally as end-to-end tests, you can run a synthetic test suite on your CI environment.
+
+Elastic's synthetics runner can output results in a few different formats, including JSON and JUnit
+(the standard format supported by most CI platforms).
+If any of your journeys fail, it will yield a non-zero exit code, which most CI systems pick up as a failure. 
```

Comment:
Nit (unclear `it`, it seems that `journeys` yield a non-zero exit code, not the agent):

```suggestion
If any of your journeys fail, the synthetics agent it will yield a non-zero exit code, which most CI systems pick up as a failure by default.
```